lighthouse-core/audits/byte-efficiency/unused-javascript.js

```diff
@@ -11,10 +11,10 @@ const JsBundles = require('../../computed/js-bundles.js');
 const i18n = require('../../lib/i18n/i18n.js');
 
 const UIStrings = {
-  /** Imperative title of a Lighthouse audit that tells the user to remove JavaScript that is never evaluated during page load. This is displayed in a list of audit titles that Lighthouse generates. */
-  title: 'Remove unused JavaScript',
-  /** Description of a Lighthouse audit that tells the user *why* they should remove JavaScript that is never needed/evaluated by the browser. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
-  description: 'Remove unused JavaScript to reduce bytes consumed by network activity. ' +
+  /** Imperative title of a Lighthouse audit that tells the user to reduce JavaScript that is never evaluated during page load. This is displayed in a list of audit titles that Lighthouse generates. */
+  title: 'Reduce unused JavaScript',
+  /** Description of a Lighthouse audit that tells the user *why* they should reduce JavaScript that is never needed/evaluated by the browser. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
+  description: 'Reduce unused JavaScript to reduce bytes consumed by network activity. ' +
```

Comment:
FWIW both of these descriptions become slightly repetitive "Reduce...to reduce...". Maybe the descriptions can be tweaked more